airflow/models/taskmixin.py

```diff
@@ -165,6 +182,11 @@ class DAGNode(DependencyMixin, metaclass=ABCMeta):
     def node_id(self) -> str:
         raise NotImplementedError()
 
+    @abstractmethod
+    def add_to_taskgroup(self, task_group: TaskGroup) -> None:
+        """Add the task to the given task group."""
+        raise NotImplementedError()
```

Comment:
Do we need to implement this? I think the parent implementation (which also raises `NotImplementedError`) would be inherited

Reply:
Was following what was done for root and leaves, removing it worked